### ISSUE TITLE
Add method to dump debug information

### ISSUE THREAD

**Issue by aidanholm** [luakit/luakit#602]
Easy way to submit information useful for debugging. Should include at a minimum:
- luakit version
- webkit version
- Lua vs LuaJIT
- settings dump
- package.path + package.cpath
- compiler if possible

Should be accessible from command-line (`luakit --debug-info`).There's probably much more useful debugging info available from luakit after startup, however, which would be nice to include.

**Comment by hippi777** [comment]
good idea, i think compiler info can be obtained in compile time easily for luakit, and for other stuffs elf header containst this info iirc

**Comment by aidanholm** [comment]
Build details like that can be compiled into `buildopts.h`, yeah
